packages/frontity-org-theme/src/utils/index.ts

```diff
@@ -37,3 +38,39 @@ export const mixAlpha = (hex: string, alpha: number) => {
     ${(1 - alpha) * 255 + alpha * Number(`0x${match[3]}`)}
   )`;
 };
+
+/*
+ *
+ *  React hook that tracks state of a CSS media query. It's based on this
+ *  library: https://github.com/streamich/use-media
+ *
+ */
```

Comment:
I would write this in TSDoc format:

```suggestion
/**
 * React hook that tracks state of a CSS media query. It's based on this
 * library: https://github.com/streamich/use-media.
 *
 * @param query - Media query in string format.
 * @returns Boolean value indicating if the media query has matched.
 */
```